### PR TITLE
Use Scout APM

### PR DESCRIPTION
**Pull request by sburba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,13 @@ heroku plugins:install @heroku-cli/plugin-manifest
 
 # Create staging environment
 heroku apps:create ttbud-staging --manifest --remote staging
+heroku config:set ENVIRONMENT=staging --remote prod
```
Comment (sjohnson31) [inline README.md:42]: This seems wrong.

Comment (sjohnson31) [reply README.md:42]:
```suggestion
heroku config:set ENVIRONMENT=staging --remote staging
```

File: api/src/scout_key_censor.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+_scout_thread_state = threading.local()
+_scout_thread_state.logged_key = False
+
+
+class ScoutKeyCensor(logging.Filter):
+    """
+    Filter that stops scout_apm from logging it's API key to the logs. :(
+
+    Should be removed after this issue is resolved:
+    https://github.com/scoutapp/scout_apm_python/issues/297
```
Comment (sburba) [inline api/src/scout_key_censor.py:14]: this issue is resolved, so we can delete this! :)

File: api/src/game_state_server.py [hunk 3]
```diff
@@ -151,102 +166,110 @@ async def process_updates(
...
                             MessageContents(
-                                'error', 'That position is occupied, bucko', request_id,
+                                'error',
+                                'Cannot delete token because it does not exist',
+                                request_id,
+                            ),
+                        )
+                elif action == 'ping':
+                    try:
+                        ping = Ping(**data)
```
Comment (sjohnson31) [inline api/src/game_state_server.py:250]: We're using dacite.from_dict for this now.

File: api/src/scout_key_censor.py (hunk 2 above)
Comment (sjohnson31) [reply api/src/scout_key_censor.py:14]: Nice.

File: api/src/game_state_server.py (hunk 3 above)
Comment (sjohnson31) [reply api/src/game_state_server.py:250]:
```suggestion
                        ping = from_dict(data_class=Ping, data=data)
```

**Pull request merged**